models/DataObject/QuantityValue/Service.php

```diff
@@ -96,14 +96,14 @@ public static function getQuantityValueUnitsTable(): ?array
                 $table = Cache\RuntimeCache::get(Unit::CACHE_KEY);
             }
 
-            if (!is_array($table)) {
+            if (!\is_array($table)) {
```

Comment:
What is the difference between is_array and \is_array?

Reply:
Is already used before in QuantityValueRange.php

https://www.deviaene.eu/articles/2023/why-prefix-php-functions-calls-with-backslash/
To me it sounds like \is_array is a bit faster

Reply:
Thanks for the article. So I did some digging and indeed it seems like a micro optimization.
Since actually we are trying to avoid \ also with other stuff like exceptions etc. we could start importing them. 

Like `use function in_array;`
Wdyt? I honstely do not feel like reintroducing backslashes :)

Reply:
@mattamon Done in https://github.com/pimcore/pimcore/pull/17190/commits/f8c272a13e6a4cced2dc6bd5a0f553740de1d423